src/ui/localization.ts

```diff
@@ -255,7 +262,7 @@ const localizeInit = () => {
                     'file.save': 'Save',
                     'file.save-as': 'Save As...',
                     'file.publish': 'Publish...',
-                    'file.export': 'Export',
+                    'file.export': 'Export...',
                     'file.export.ply': 'PLY (.ply)',
```

Comment:
Can all the `file.export` strings go now?

Reply:
addressed in commit 20f481d35aeae200de9f01bc11a111060b6a809d